compiler/optimizations/loopInvariantCodeMotion.cpp

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright 2017 Advanced Micro Devices, Inc.
```

Comment:
@tmacd9 ignorant question on my part -- how do copyrights work with our license/CLA? Do we/cray have sole copyright, or can AMD add theirs?

If they can add theirs is the right place to add it?